smart-gate/readme.md

```diff
@@ -3,7 +3,9 @@
 ## Introduction
 This guide will walk you through the process of building contracts for the Smart Gate, deploying them into an existing world running in Docker, and testing their functionality by executing scripts.
```

Comment:
no that we have the new flow for local deployment we might need changes to readme where they can just run 
`pnpm run dev` instead of having to deploy to the existing world. 
also this section needs update  ### Step 2: Mock data for the existing world **(Local Development Only)**

Reply:
and add command to run the tests instead of having to mock data